quiche/src/ffi.rs

```diff
@@ -1091,22 +1091,25 @@ pub struct Stats {
     lost_bytes: u64,
     stream_retrans_bytes: u64,
     paths_count: usize,
-    peer_max_idle_timeout: u64,
-    peer_max_udp_payload_size: u64,
-    peer_initial_max_data: u64,
-    peer_initial_max_stream_data_bidi_local: u64,
-    peer_initial_max_stream_data_bidi_remote: u64,
-    peer_initial_max_stream_data_uni: u64,
-    peer_initial_max_streams_bidi: u64,
-    peer_initial_max_streams_uni: u64,
-    peer_ack_delay_exponent: u64,
-    peer_max_ack_delay: u64,
-    peer_disable_active_migration: bool,
-    peer_active_conn_id_limit: u64,
-    peer_max_datagram_frame_size: ssize_t,
     paths: [PathStats; 8],
 }
 
+pub struct TransportParams {
```

Comment:
You also need to update [quiche.h](https://github.com/cloudflare/quiche/blob/master/quiche/include/quiche.h) with both type (something like `quiche_transport_params`) and function.

Reply:
Oops, updated